website/layouts/partials/_default/sidemenu.html

```diff
@@ -1,68 +1,149 @@
+      </label>
+        
+      {{ range .Pages }}
+
+      <div class="list-group list-group-flush">
+        <a class="list-group-item list-group-item-action list-group-item-thanos py-1" href="{{ .Permalink }}">{{ .Title }}</a>
+      </div>
+
+      {{ end }}
+ 
     </div>
+    {{ end }}
+    {{ end }}
+
+    {{ range .Pages.GroupByParam "menu" }}
+    {{ if eq .Key "contributing"}}
     <div class="sidemenu-thanos">
-        <input type="checkbox" id="menuinput-contributing" class="menuinput"{{if eq .Section "contributing"}} checked{{end}} />
-        <label for="menuinput-contributing" class="menulabel-thanos">
-            <h5 class="mt-3">Contributing</h5>
-            <img class="caret-expand mt-3" src="/expand.svg" alt="expand" />
-        </label>
-        <div class="list-group list-group-flush">
-            {{ range .Site.Menus.contributing }}
-                <a class="list-group-item list-group-item-action list-group-item-thanos py-1" href="{{ .URL }}">{{ .Title }}</a>
-            {{ end }}
-        </div>
+      <input type="checkbox" id="menuinput-thanos" class="menuinput" {{if eq .Key "contributing"}} checked {{end}}/>
+      <label for="menuinput-thanos" class="menulabel-thanos">
+        <h5 >{{ .Key }}</h5>
+        <img class="caret-expand" src="/expand.svg" alt="expand" />
+      </label>
+        
+      {{ range .Pages }}
+
+      <div class="list-group list-group-flush">
+        <a class="list-group-item list-group-item-action list-group-item-thanos py-1" href="{{ .Permalink }}">{{ .Title }}</a>
+      </div>
+
+      {{ end }}
+ 
     </div>
+    {{ end }}
+    {{ end }}
+
+    {{ range .Pages.GroupByParam "menu" }}
+    {{ if eq .Key "proposals"}}
     <div class="sidemenu-thanos">
-        <input type="checkbox" id="menuinput-proposals" class="menuinput"{{if eq .Section "proposals"}} checked{{end}} />
-        <label for="menuinput-proposals" class="menulabel-thanos">
-            <h5 class="mt-3">Proposals</h5>
-            <img class="caret-expand mt-3" src="/expand.svg" alt="expand" />
-        </label>
-        <div class="list-group list-group-flush">
-            {{ range .Site.Menus.proposals }}
-                <a class="list-group-item list-group-item-action list-group-item-thanos py-1" href="{{ .URL }}">{{ .Title }}</a>
-            {{ end }}
-        </div>
+      <input type="checkbox" id="menuinput-thanos" class="menuinput" {{if eq .Key "proposals"}} checked {{end}}/>
+      <label for="menuinput-thanos" class="menulabel-thanos">
+        <h5 >{{ .Key }}</h5>
+        <img class="caret-expand" src="/expand.svg" alt="expand" />
+      </label>
+        
+      {{ range .Pages }}
+
+      <div class="list-group list-group-flush">
+        <a class="list-group-item list-group-item-action list-group-item-thanos py-1" href="{{ .Permalink }}">{{ .Title }}</a>
+      </div>
+
+      {{ end }}
+ 
     </div>
+    {{ end }}
+    {{ end }}
+
+  {{ end }}
+  {{ end }}
 </nav>
+{{ end }}
+
```

Comment:
Why this is needed below? Let's make it clean (:

Reply:
Done